util/libcrypto.num

```diff
@@ -4682,7 +4682,7 @@ ERR_set_error                           ?	3_0_0	EXIST::FUNCTION:
 ERR_vset_error                          ?	3_0_0	EXIST::FUNCTION:
 X509_get0_authority_issuer              ?	3_0_0	EXIST::FUNCTION:
 X509_get0_authority_serial              ?	3_0_0	EXIST::FUNCTION:
-EC_GROUP_new_ex                         ?	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_3_0,EC
+EC_GROUP_new_ex                         ?	3_0_0	NOEXIST::FUNCTION:DEPRECATEDIN_3_0,EC
```

Comment:
@t8m well above is `make update` result but I don't think it really makes sense?

Reply:
If you remove manually the line and re-run make update, it should not reappear.

Reply:
perfect ty that works.

I'll do the `pod` and `CHANGES.md` edits now, then squash and this one should be ready.